recipes/handcalcs/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "handcalcs" %}
+{% set version = "1.4.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/handcalcs-{{ version }}.tar.gz
+  sha256: 273baafb1f017df44061740755d2a00e987ead53871ab3bb0bab4aac0358675c
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - flit
+  run:
+    - innerscope >=0.2.0
+    - jupyterlab >=3.0.0
```

Comment:
Upstream only lists this as a "dev" dependency? Is that incorrect?

Reply:
I used grayskull to create the recipe - but you are right it should work without `jupyterlab` - I am testing it right now.